README.md

```diff
@@ -2,30 +2,86 @@ Core ML Community Tools
 =======================
 
 Core ML community tools contains all supporting tools for Core ML model
-conversion and validation. This includes scikit-learn, LIBSVM, Caffe,
-Keras and XGBoost.
+conversion, editing and validation. This includes scikit-learn, LIBSVM, Caffe,
+Keras, Tensorflow and XGBoost.
 
-coremltools 3.0
----------------
-[Release notes](https://github.com/apple/coremltools/releases/)
+# Table of contents
 
+* [Release information](#Release-Information)
+* [Neural network conversion guide](#Neural-network-conversion-guide)
+* [Core ML specification](#Core-ML-specification) 
+* [coremltools documentation](#coremltools-documentation-and-examples)
+* [Installation](#Installation)
 
-👍👎 Please take this quick poll and let us know how you liked this release, [here](https://github.com/apple/coremltools/blob/master/release-feedback.md)!
+
+
+## Release information
+
+Latest version of coremltools is 3.1  
 
 ```shell
 # Install using pip
-pip install coremltools==3.0
+pip install coremltools==3.1 
+# or
+pip install -U coremltools
 ```
 
-API
----
-- [Example Code Snippets](docs/APIExamples.md)
-- [coremltools Documentation](https://apple.github.io/coremltools)
-- [Core ML Specification Documentation](https://apple.github.io/coremltools/coremlspecification/)
-- [IPython Notebooks](https://github.com/apple/coremltools/tree/master/examples)
+For the latest changes please see the [release notes](https://github.com/apple/coremltools/releases/)
+
+**10/31/2019**: Released coremltools 3.1. Added support for models defined via the tf.keras API in TF 2.0. There are a few known issues, please see the release notes.   
+**10/4/2019**: Non-beta version of coremltools 3.0 released. 
+
+
+## Neural network conversion guide
+
+There are several `converters` available to translate neural networks trained in various frameworks into the Core ML model format.
+Following formats can be converted to the Core ML `.mlmodel` format through the coremltools python package (this repo):
+
+- Caffe V1 (.prototxt, .caffemodel format)
+- Keras API (2.3) (.h5 format)
+- Tensorflow (TF 1.13+) (.pb frozen graph def format)
+- tf.keras API (only for TF 2.0, .h5 and saved-model formats)
+
```

Comment:
I'd just use `TensorFlow 1.x` and `TensorFlow 2.x`. The `tf.keras` is a bit more details.

Reply:
Okay

Reply:
changed it